test/parallel/test-fs-fchmod.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+
+// This test ensures that input for fchmod is valid, testing for valid
+// inputs for fd and mode
+
+// Check input type
+['', false, null, undefined, {}, [], Infinity, -1].forEach((i) => {
+  common.expectsError(
+    () => fs.fchmod(i),
+    {
+      code: 'ERR_INVALID_ARG_TYPE',
+      type: TypeError,
+      message: 'The "fd" argument must be of type integer'
+    }
+  );
+  common.expectsError(
+    () => fs.fchmodSync(i),
+    {
+      code: 'ERR_INVALID_ARG_TYPE',
+      type: TypeError,
+      message: 'The "fd" argument must be of type integer'
+    }
+  );
+
+  common.expectsError(
+    () => fs.fchmod(1, i),
+    {
+      code: 'ERR_INVALID_ARG_TYPE',
+      type: TypeError,
+      message: 'The "mode" argument must be of type integer'
+    }
+  );
+  common.expectsError(
+    () => fs.fchmodSync(1, i),
+    {
+      code: 'ERR_INVALID_ARG_TYPE',
+      type: TypeError,
+      message: 'The "mode" argument must be of type integer'
+    }
+  );
+});
+
+// Check for mode values range
+const modeUpperBoundaryValue = 0o777;
+common.mustCall(() => fs.fchmod(1, modeUpperBoundaryValue));
+common.mustCall(() => fs.fchmodSync(1, modeUpperBoundaryValue));
```

Comment:
Is it me or something is missing? The function returned by `common.mustCall` is never called, so I expect the test to fail.

Reply:
Good point. And neither of these two entries should require being wrapped in a mustCall.